website/docs/source/docs/boxes/base_macos.html.md

```diff
@@ -0,0 +1,172 @@
+
+    To Say:
+
+    ``` bash
+    %admin ALL=(ALL) NOPASSWD: ALL
+    ```
+
+    Now, you should now be able to run sudo without password.
+
+* Install Parallels Tools inside the macOS virtual machine.
+* Boot macOS virtual machine and enable Remote Login (System Settings > General > Sharing > Enable Remote Login). Don't forget to give “Full disk access to users”, do allow “All users” (in the same settings, press ![info](/images/info_32.png) button)
+![Parallels Desktop](/images/allow_sharing.gif)
+* Restart macOS virtual machine for the Remote Login to take effect.
+
+## SSH Keys
+
+We will need to create a new SSH keypair for Vagrant to use to communicate
+
+* Create an ssh public-private key from the host.
+  * Start Terminal on the host side.
+  * Execute the following command:
+
+        ``` bash
+        $ ssh-keygen -t rsa
+        ```
+
+  * After every step press "Enter". You don't need to use a passphrase or file. As a result, you will get a similar output as in the example below:
+  ![ssh_key](/images/ssh_key.jpeg)
+  It will generate the files below in the ~/.ssh/ directory:
+  ![ssh_pair](/images/ssh_pair.png)
+* Upload the public key to macOS virtual machine from your host by executing the command below in Terminal:
+  
+    ``` bash
+    $ ssh-copy-id vagrant@<virtual machine's ipv4 address>
+    ```
+
+    <div class="alert alert-info">
+    <p>
+        <strong>Note:</strong> To check your virtual machine's IP address, go to Finder > System Settings > Network > Ethernet > IP address.
+    </p>
+    </div>
+
+## Create Base Box
+
+* In Parallels directory on your Mac create a folder (something simple like ```VagrantTest```).
+* Copy the initially created bundle of your macOS virtual machine (it has the **.macvm** extension) to the newly created folder (```VagrantTest```).
+    
+    <div class="alert alert-info">
+    <p>
+        <strong>Optional:</strong> You can rename macOS virtual machine with a unique name so that you won't get any name collision issues.
+    </p>
+    </div>
+
+  * Run Terminal on the host side and execute the following command:
+  
+    ``` bash
+    $ prlctl set "macOS 13" --name "macOS_13"
+    ```
+
+    Where the name 'macOS_13' is your virtual machine's name.
+
+    <div class="alert alert-info">
+    <p>
+        <strong>Note:</strong> Renaming the virtual machine is one of the solutions to avoid name collision. You can also remove macOS virtual machine from Control Center (Parallels icon > Control Center > Right-click on macOS 13 virtual machine > Remove 'macOS 13' > <strong>Keep Files</strong>).
+    </p>
+    </div>
+
+* Download [Vagrantfile](https://kb.parallels.com/Attachments/kcs-191881/Vagrantfile) and put it in the VagrantTest folder.
+* Download [metadata.json](https://kb.parallels.com/Attachments/kcs-191881/metadata.json) file and put it in the VagrantTest folder.
+* Copy your **private** key from ~/.ssh/ directory in VagrantTest directory and rename it to '**vagrant\_private\_key**':
```

Comment:
no need to do this if using the public pair.

Reply:
added an alert that if they choose the public vagrant key they would not need to do that part